apps/viewer/.env.development

```
@@ -0,0 +1,3 @@
+VITE_WS_RELAY_URL=ws://localhost:4444
+VITE_DEFAULT_MODEL_URL=http://localhost:5173/models/demo.ifc
+VITE_APP_NAME=IFC Live (Dev)
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_NAME key should go before the VITE_DEFAULT_MODEL_URL key

(UnorderedKey)